src/common/iframe/iframe-event-listener.ts

```diff
@@ -75,7 +78,7 @@ export default class IframeEventListener<TEventMap extends IframeEventMap<keyof
 
     @bind
     private _handleMessage(event: MessageEvent): void {
-        if ((event.origin !== this._sourceOrigin) ||
+        if ((this._sourceOrigins.indexOf(event.origin) === -1) ||
```

Comment:
🍹  can use `!this._sourceOrigins.includes(event.origin)`for better readability

Reply:
I just realised I can't do this because we don't polyfill `Array` for the NPM release. 😞